nexus/social_media/services.py

```diff
@@ -1,34 +1,213 @@
+# Standard Library
+from datetime import datetime
+
 # Third Party Stuff
 import facebook
+import requests
 from django.conf import settings
+from django.utils import timezone
+from rest_framework import status
 
 # nexus Stuff
-from nexus.base import exceptions
-# Nexus Stuff
+from nexus.base import exceptions as exc
+from nexus.social_media import tasks
 from nexus.social_media.models import Post
 
 
+def upload_image_to_linkedin(author, headers, post, linkedin):
+    """Upload image to linkedin for given post.
+
+    :param author: Owner of the post to be used for response POST data.
+
+    :param headers: HTTP headers required for successful HTTP request.
+
+    :param post: Post model instance.
+
+    :param linkedin: LINKEDIN_AUTH object taken from settings.
+
+    :returns: A valid HTTP response.
+
+    """
+    post_data_assets = {
+        "registerUploadRequest": {
+            "recipes": [
+                "urn:li:digitalmediaRecipe:feedshare-image"
+            ],
+            "owner": author,
+            "serviceRelationships": [
+                {
+                    "relationshipType": "OWNER",
+                    "identifier": "urn:li:userGeneratedContent"
+                }
+            ]
+        }
+    }
+
+    api_url_assets = f"{settings.LINKEDIN_API_URL_BASE}assets?action=registerUpload"
+
+    response_assets = requests.post(api_url_assets, json=post_data_assets,
+                                    headers=headers)
+    if response_assets.status_code == status.HTTP_200_OK:
+        json_response_assets = response_assets.json()
+        upload_url = json_response_assets.get('value')\
```

Comment:
rather than using `\` at every line, you can just move the subsequent calls in the next line.